streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamsKafkaClient.java

```diff
@@ -217,7 +217,7 @@ private String ensureOneNodeIsReady(final List<Node> nodes) {
                 }
             }
             try {
-                kafkaClient.poll(500, Time.SYSTEM.milliseconds());
+                kafkaClient.poll(0, Time.SYSTEM.milliseconds());
```

Comment:
We can update to a higher value... But what would be a good default? It's a temporary fix anyway as we replace this with AdminClient soon. \cc @guozhangwang @dguy 

Also, system tests did not complain about this.